app/controllers/course/achievements_controller.rb

```diff
@@ -3,4 +3,41 @@ class Course::AchievementsController < Course::ModuleController
 
   def index #:nodoc:
   end
+
+  def new #:nodoc:
+  end
+
+  def edit #:nodoc:
+  end
+
+  def create #:nodoc:
+    if @achievement.save
+      redirect_to(course_achievements_path(@course),
+                  notice: t('.notice', title: @achievement.title))
+    else
+      render 'new'
+    end
+  end
+
+  def update #:nodoc:
+    if @achievement.update_attributes(achievement_params)
+      redirect_to(course_achievements_path(@course),
+                  notice: t('.notice', title: @achievement.title))
+    else
+      render 'edit'
+    end
+  end
+
+  def destroy #:nodoc:
+    if @achievement.destroy
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.